draft-ietf-tls-esni.md

```diff
@@ -944,13 +944,11 @@ It then computes a string
     accept_confirmation =
         Derive-Secret(Handshake Secret,
                       "ech accept confirmation",
-                      ClientHelloInner..ServerHelloECHConf)
+                      ClientHelloInner || ServerHelloECHConf)
 ~~~
 
 where Derive-Secret and Handshake Secret are as specified in {{RFC8446}},
-Section 7.1 and ClientHelloInner..ServerHelloECHConf refers to the sequence of
-handshake messages beginning with the first ClientHello and ending with
-ServerHelloECHConf. Finally, the backend server constructs its ServerHello
+Section 7.1. Finally, the backend server constructs its ServerHello
```

Comment:
If we revert the above change, don't forget to revert this change as well.

Reply:
Done!